sktime/forecasting/model_evaluation/tests/test_evaluate.py

```diff
@@ -77,10 +78,14 @@ def _check_evaluate_output(out, cv, y, scoring):
 @pytest.mark.parametrize("step_length", TEST_STEP_LENGTHS)
 @pytest.mark.parametrize("strategy", ["refit", "update"])
 @pytest.mark.parametrize(
-    "scoring", [MeanAbsolutePercentageError(symmetric=True), MeanSquaredError()]
+    "scoring",
+    [
+        MeanAbsolutePercentageError(symmetric=True),
+        MeanSquaredError(),
```

Comment:
I suggest to remove this to speed up tests, since MSE doesn't test anything new given MAPE and MASE. Perhaps add a comment why we use MAPE and MASE (i.e. one with and one without `y_train`)

Reply:
Or MSE instead of MAPE as you do below

Reply:
@mloning I've got that done and added the comment. Forgot to remove MeanSquaredError() after changing up the testing for MASE after removing that validation. 

Hopefully this is looking good to go.